File: BackEnd/Service/Models/RecruiterModel.cs
using Data.Entities;

namespace Service.Models;

public partial class RecruiterModel
{
    public Guid RecruiterId { get; set; }
    public string UserId { get; set; } 
    public virtual WebUser User { get; set; } 
    public Guid CompanyId { get; set; }
    public virtual CompanyModel Company { get; set; } 
    public bool IsDeleted { get; set; } = false;
    public bool IsActived { get; set; } = false;
}

File: FrontEnd/src/app/data/recruiter/recruiter.model.ts
import { WebUser } from "../authentication/web-user.model";
import { Company } from "../company/company.model";

export class Recruiter {
	recruiterId?: string;
	userId?: string;
	user?: WebUser;
	departmentId?: string;
	companyId?: string;
	company?: Company;
	isDeleted?: boolean;
	isActived?: boolean;
}


export class RecruiterAddModel {
	userId?: string;
	companyId?: string;
	isActived?: boolean;
	isDeleted?: boolean;
}
